swap/swap.go

```diff
@@ -80,8 +85,42 @@ func NewParams() *Params {
 	}
 }
 
+// newLogger returns a new logger
+func newLogger(logpath string) log.Logger {
+	swapLogger := log.New("swaplog", "*")
```

Comment:
is this the standard way of segregating logs?

 the `*` seems a bit like clutter, although i suppose we need to add something that says `swap` to the context